### ISSUE TITLE
Touch optimized and customizable Webfronmtend

### ISSUE THREAD

**Issue by splitti** [jaedb/Iris#695]
**Is your feature request related to a problem? Please describe.**
I really like IRIS! I try to create a Box for my small kids. One is not able to read, so it would be perfect, if he gets an album view and play view only.

**Describe the solution you'd like**
It would be nice to disable some entrys on the bar (no search, no this and that...)

I just want a few points in the bar, Settings, my Albums, my Artists and my Playlists from Spotify. Maybe you have an idea, how to port IRIS for that, like a second IRIS sandbox...

**Comment by jaedb** [comment]
The first port of call would be to visualise how you anticipate this working, and to better understand the wider use case. Some key considerations I would also suggest exploring:
1. Who is it for (beyond your specific use case)
2. How would you describe it's purpose to a new user
3. What features _must_ it have (as opposed to 'nice to haves') 

The reality is that every feature request requires a balance between effort required and benefit it provides. Unfortunately many requests just can't be justified. It is important that my personal time spent developing Iris is on functionality that delivers as much value to as many users as possible.

Basically: propose how this would work and what it would feel like, and we can go from there :-)
